lib/test_nplusone.py

```diff
@@ -2,131 +2,131 @@
 import time
 import tempfile
 
-
 __doc__ = """ Add nfiles to a directory and check consistency.
 """
 
-from smashbox.utilities import *
 from smashbox.utilities.hash_files import *
 from smashbox.utilities.monitoring import push_to_monitoring
+import platform
+import logging
 
-nfiles = int(config.get('nplusone_nfiles',10))
-filesize = config.get('nplusone_filesize',1000)
+nfiles = int(config.get('nplusone_nfiles', 20))
+filesize = config.get('nplusone_filesize', 10000)
 
 # optional fs check before files are uploaded by worker0
-fscheck = config.get('nplusone_fscheck',False)
+fscheck = config.get('nplusone_fscheck', False)
```

Comment:
This kind of change (added whitespaces!) should never be part of pull request. It makes it very difficult to understand which changes are actually relevant among all the changes.